lib/net/ssh/config.rb

```diff
@@ -155,6 +155,9 @@ def load(path, host, settings = {}, base_dir = nil)
             %w[proxyjump proxycommand].each do |proxy_key|
               if (proxy_value = settings.delete(proxy_key))
                 settings['proxy'] ||= [proxy_key, proxy_value]
+                if proxy_key == 'proxyjump' and proxy_value == 'none'
+                  settings.delete('proxy')
+                end
```

Comment:
Need to leave the setting in place so later ProxyJump / ProxyCommand directives don't override it.